unittests/CMakeLists.txt

```
@@ -64,7 +64,7 @@ macro (create_tests_with_backend NVQIR_BACKEND EXTRA_BACKEND_TESTER)
     nvqir-${NVQIR_BACKEND} nvqir  
     cudaq fmt::fmt-header-only
     cudaq-platform-default
-    cudaq-builder
+    cudaq-builder 
```

Presumably this new trailing space isn't really needed?
```suggestion
    cudaq-builder
```